routes/users.js

```diff
@@ -1,9 +1,62 @@
 var express = require('express');
 var router = express.Router();
+var passport = require('passport');
+const db = require('../models');
+
+//This file needs to run and have nothing to export
+require('../controllers/user.controller');
 
 /* GET users listing. */
 router.get('/', function (req, res, next) {
     res.send('respond with a resource');
 });
 
+router.get('/auth/facebook', passport.authenticate('facebook', { scope : ['email']}));
+
+router.get('/auth/facebook/callback',
+    passport.authenticate('facebook', {
+      successRedirect: 'success',
+      failureRedirect: 'fail'
+    }));
+
+router.get('/auth/facebook/fail', (req,res) => {
+  // Failed login landing page
+  res.status(400).send('Something went wrong...');
+});
+
+router.get('/auth/facebook/success', (req,res) => {
+  // Successfull login landing page
+    res.redirect('../../demo');
+
+    // TODO
+    // To integrate with FE we want to change to answer to something like this
+    // res.status(200).send('ok');
+});
+
+router.get('/logout', (req, res) => {
+    if(!req.user) {
```

Comment:
why not just use the req.logout like in their example?

Reply:
which example?
comment it out in a branch designated for connecting FE and BE

Reply:
the logic you've implemented is pretty much what passport's `request.logout()` does, according to their official site, have a look and tell me what you think:
http://www.passportjs.org/docs/logout/